### ISSUE TITLE
Athlete Settings Across Different Computers

### ISSUE THREAD

**Issue by A1Dan75** [thomaschampagne/elevate#767]
I'm not sure if I'm missing something or this is just the intended behaviour, but is there any way for my athlete settings (and hence my fitness trend etc) to be saved across all computers on which I use Elevate?  I set-up a load of Threshold / FTP etc settings on my home machine, but when I go on my work machine they are not there.

**Comment by zeair** [comment]
+1 for this! 

I do not expect synchronization - this would require infrastructure on your side but dummy import export possibility would be great! 

I already tried to look into checked local storage, session storage and indexdb - where do you store athlete configuration? 

I could then make chrome extension to import/export.

**Comment by greglondon** [comment]
You can export / import zones (HR, speed, cadence, etc.). I think other athlete settings you need to manually enter on the other computer. I've done it and works completely fine on both (home and work).

**Comment by thomaschampagne** [comment]
Backup of activities already export your dated athlete settings. Open that file scroll down to see them.

They are restored as well. 

I'm of course aware of all this but indeed a "real" sync over the network should imply servers cost and important data storage. I can't pay that at the moment with a free and opensource project. Even with donations i receive :(.

**Comment by A1Dan75** [comment]
No, of course.  I hadn't really thought about the need for infrastructure on your side to be able to run things that way.

On a related note, I've come across an issue which I wonder if you're familiar with?:

When I enter my settings on a new computer and attempt to synchronise, it fails to pull my activities in and I get a partial fitness trend graph.  If I then go back into Strava my activities do not load properly, the speed etc graphs do not appear and the map doesn't show my route.  It seems like maybe Strava is overloaded and not responding, or possibly they are blocking my access?

Actually, I just tried this again and it's changed slightly, I get this message when syncing:

Synchronization stopped. Don't worry.
The sync had to pause due to Strava servers errors or potential overload. To avoid this, please relaunch sync in few hours. It will resume where it stopped. Already synced activities will not be calculated.

**Comment by A1Dan75** [comment]
Sorry, I'm probably being very thick here... where is the "Backup of activities" function you mention?

**Comment by thomaschampagne** [comment]
In the app top right and corner

**Comment by thomaschampagne** [comment]
Here is the thing: https://twitter.com/champagnethomas/status/1095743506193305600

And this message is expected with the new patch. In you case you should "clear and resync" because you goes wrong last week (before patch) ! You will be able to sync ~400 activities each times and wait hours between (i don't known how many yet). Strava has implemented some limits per user on their servers.

**Comment by A1Dan75** [comment]
Thanks Thomas, that's much as I though, Strava must be getting fed up of all the requests... your app must be too popular!

Found the backup, I was looking under the three dots menu, but it's the sync status you need to click on, isn't it!

**Comment by zeair** [comment]
are you kidding me? new version ereased all my dated athlete settings! I got about 30 entries!

**Comment by A1Dan75** [comment]
Yeah, it's wiped mine and says it needs to sync from scratch again.

**Comment by marguslt** [comment]
Same issue here, athlete settings were wiped. Although there might be something else to it -  can't be too sure anymore, but I believe I noticed it only after I had restarted Chrome on my Win box. It seems that Chrome had an update last week (73.0.3683.86 - 26.03.2019) and there's a good chance that this was the first time new version was run. 
Another thing - when I start a new sync, that window just closes after few minutes, normally it would stay open with _retry-after-few-hours_ message. I'll try with fresh install soon, but any way to debug this? 

On my Linux laptop I had disabled google repositories, so Chrome version was rather old, there everything kept working after Elevate update, all athlete settings stayed in place an noticed no other issues.
